dashboard/pages/data_projection/stages/s2_projection_display.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+from dash import dash_table, dcc, html
+from dash.dash_table.Format import Format, Scheme
+
+PROJECTION_DISPLAY_STAGE = html.Div(
+    [
+        html.Div(
+            className="row",
+            children=[
+                html.Div(
+                    [html.H5("Compounds Data:")],
+                    className="col-md-6",
+                ),
+                html.Div(
+                    [
+                        html.Div(
+                            className="row",
+                            children=[
+                                html.Div(
+                                    [
+                                        html.Div(
+                                            id="projection-method-selection-box",
+                                            children=[
+                                                dcc.Dropdown(
+                                                    options={},
+                                                    placeholder="Select a method",
+                                                    disabled=True,
+                                                ),
+                                            ],
+                                        )
+                                    ],
+                                    className="col",
+                                ),
+                                html.Div(
+                                    [
+                                        html.Div(
+                                            id="projection-attribute-selection-box",
+                                            children=[
+                                                dcc.Dropdown(
+                                                    options={},
+                                                    placeholder="Select an attribute",
+                                                    disabled=True,
+                                                ),
+                                            ],
+                                        )
+                                    ],
+                                    className="col",
+                                ),
+                            ],
+                        ),
+                    ],
+                    className="col-md-6",
+                ),
+            ],
+        ),
+        html.Div(
+            className="row mt-3",
+            children=[
+                html.Div(
+                    children=[
+                        html.Div(
+                            id="projection-table",
```

Comment:
As far as I remember, the table automatically adjusted for the compounds currently visible on the plot. It doesn't work now, not sure if it was intended

Reply:
There are couple of things tbd in order to match the previous version:
- dynamic data table connected with plot 
- add control values 
- loading sign (and this one to be set everywhere imo)
- ACTIVATION/INHIBITION choice for every experiment

Will add it to the issues & can take care of it, just wanted to provide the basic version; I can turn it into a draft and work on it further too.

Reply:
ok sure, we may work on that later